site/gulpfile.js

```diff
@@ -22,6 +23,11 @@ const RESIZE_OPTS = {
     errorOnUnusedConfig: false
 };
 
+let manifest = {
```

Comment:
I think we need to determine this based on environment 🤔 we might run into an issue where the files aren't built but the manifest still exists 😵

Reply:
That's why I added the `clean:rev` in the first place. What do you think about keeping the said rule?

Reply:
It should be good by now